### PR TITLE
Add emoji picker

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
markdown editor

### Description
This PR adds an emoji / fork awesome icon picker to the tool bar of the editor
### Steps

- [x] added implementation
- [x] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### Related Issue(s)
#323

### CODE REVIEW

**Comment by DerMolly** [comment]
This has three hurdles currently that need to be overcome, before this can be merged.
1. merge #302 
2. make it possible to close the picker by clicking elsewhere
3. show all the fork awesome icons in the picker it self

1. & 2. seem rather easy

Not sure how to tackle 3. currently. The Picker want's the custom icons to be images (or a sprite sheet [maybe fork awesome provides something like that]) to put them in via css `background-image`, but we only have a font.

File: src/components/editor/editor-window/tool-bar/emoji-picker/emoji-picker.tsx [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
-          emoticons: [],
-          keywords: ['fork awesome'],
-          imageUrl: 'https://github.githubassets.com/images/icons/emoji/octocat.png',
-          customCategory: 'ForkAwesome'
-        }))
-        }
-      />
+      <div ref={pickerRef}>
+        <NimblePicker
+          ref={pickerRef}
```
Comment (ErikMichelson) [inline src/components/editor/editor-window/tool-bar/emoji-picker/emoji-picker.tsx:27]: This looks as if the `div` and the `NimblePicker` both have the same reference object linked?

Comment (DerMolly) [reply src/components/editor/editor-window/tool-bar/emoji-picker/emoji-picker.tsx:27]: yeah the `NimblePicker` should not have the ref as that did not work and was my initial try

Comment (DerMolly) [reply src/components/editor/editor-window/tool-bar/emoji-picker/emoji-picker.tsx]: fixed

**Comment by ErikMichelson** [comment]
The idea is to get this merged with the forkawesome icon as a placeholder instead of the correct icons.
Before merge we just need to ensure the positioning of the picker is correct. @mrdrogdrog could you have a look at this?

**Comment by mrdrogdrog** [comment]
Sure

**Pull request merged**